package kairos

class ErrorService {

    static transactional = true

    def generateError() {
		throw new Exception("Excepcion provocada")
    }
}
